docs/payments-refunds/error-codes.md

```diff
@@ -47,7 +47,7 @@ If any client errors occur, our endpoints return an error object describing the
 | unauthorized_access   | Authorization error occurred. |
 | availability_duration_expired   | It is no longer possible to refund this payment.|
 
-### HTTP 400 Bad Request
+### HTTP 400 bad request
```

Comment:
Was this changed to all lower cases on purpose?

Reply:
We did in other places, but here it shouldn't be. I'm change back.